data/_ontologies/Resources.csv

```diff
@@ -1,2 +1,3 @@
-id,name,type
-RD3,Solve-RD RD3,Other type
\ No newline at end of file
+id,name,type,type other,acronym,cohort type,website,start year,end year
+ROR,Research Organization Registry,"Data source,Other type",Ontology,ROR,,https://ror.org/,,
+RD3,Solve-RD,"Data source,Rare disease,Registry",,,Clinical cohort,https://solve-rd.eu,2018,2022
```

Comment:
this file should not exist in my mind. We don't have an ontology called Resources and we should not have it to not create confusion with the Resources table in our data model.